sys-apps/man-db/man-db-2.7.6.1-r3.ebuild

```diff
@@ -105,4 +113,9 @@ pkg_postinst() {
 		einfo "Rebuilding man-db from scratch with new database format!"
 		mandb --quiet --create
 	fi
+
+	if use systemd; then
```

Comment:
That's not what [Mike's comment](https://bugs.gentoo.org/show_bug.cgi?id=622236#c1) was about.